showcase.json

```diff
@@ -3296,5 +3296,29 @@
     "github": "https://github.com/QuezarHQ/pingICP",
     "twitter": "https://twitter.com/quezarHQ",
     "submittableId": "41431540"
+  },
+  {
+  "name": "Alphaday",
+  "description": "Alphaday is a powerful crypto workflow aggregator that pulls in news, on-chain data, social media, web3 services and much more into one customizable UI.",
+  "website": "https://app.alphaday.com/b/dfinity/",
+  "logo": "/img/showcase/alphaday_logo.webp",
+  "screenshots": [
+    "/img/showcase/alphaday_screenshot.webp"
```

Comment:
Can you please add a screenshot file to this path?

Reply:
Done